### PR TITLE
Fix 1276

### PR DESCRIPTION
**Pull request by rgrinberg**
I suppose this bug as present before #1276, but the action field certainly helped discover it.

### CODE REVIEW

**Comment by rgrinberg** [comment]
@diml would you mind accepting the corrections for this test and pushing the result? I cannot update the output-obj test.

File: src/simple_rules.ml [hunk 1]
```diff
@@ -102,6 +102,7 @@ let alias sctx ?extra_bindings ~dir ~scope (alias_conf : Alias_conf.t) =
         ~f:Dune_file.Dep_conf.remove_locs alias_conf.deps
     , Option.map ~f:(fun (_loc, a) -> Action.Unexpanded.remove_locs a)
         alias_conf.action
+    , Option.map extra_bindings ~f:Pform.Map.to_stamp
```
Comment (rgrinberg) [inline src/simple_rules.ml]: Perhaps more correct would be to take the subset of extra_bindings that is actually used by the alias. I'm not sure if this extra is justified though (@diml ?)

**Review by emillon — COMMENTED** [review]
:+1: this can go in a point release

**Comment by diml** [comment]
I updated the output-obj test

File: src/simple_rules.ml (hunk 1 above)
Comment (diml) [reply src/simple_rules.ml]: I don't think so. AFAICS, the only extra binding is the `test` variable. If it's not used, then I don't know what the test is doing exactly...

**Review by ghost — APPROVED** [review]

**Pull request merged**